qiskit_alice_bob_provider/local/translation_plugin.py

```diff
@@ -262,7 +115,7 @@ def pass_manager(
         }
 
         # Use as basis the same pass manager as the LocalStatePreparationPlugin
-        pm = LocalStatePreparationPlugin().pass_manager(
+        pm = StatePreparationPlugin().pass_manager(
             pass_manager_config=pass_manager_config,
             optimization_level=optimization_level,
         )
```

Comment:
I find it confusing to have a shared `translation_plugin` for local and remote and a second one with the same name specifically for local. Could we think of a different name for this file like `local_translation.py`?

Reply:
My bad this is addressed in a later commit actually, nice :+1: